### PR TITLE
Updated metal-go client for sub-commands Users

### PR DESCRIPTION
**Pull request by codinja1188**

### CODE REVIEW

File: internal/users/retrieve.go [hunk 1]
```diff
@@ -44,22 +45,22 @@ func (c *Client) Retrieve() *cobra.Command {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			cmd.SilenceUsage = true
 			var err error
-			var user *packngo.User
+			var user *metal.User
 			if userID == "" {
-				user, _, err = c.Service.Current()
+				user, _, err = c.Service.FindCurrentUser(context.Background()).Include(nil).Exclude(nil).Execute()
```
Comment (ctreatma) [inline internal/users/retrieve.go]: You can leave out the `Include` and `Exclude` calls here

File: internal/users/retrieve.go [hunk 2]
```diff
@@ -44,22 +45,22 @@ func (c *Client) Retrieve() *cobra.Command {
...
 				if err != nil {
 					return fmt.Errorf("Could not get current User: %w", err)
 				}
 			} else {
-				user, _, err = c.Service.Get(userID, c.Servicer.ListOptions(nil, nil))
+				user, _, err = c.Service.FindUserById(context.Background(), userID).Include(nil).Exclude(nil).Execute()
```
Comment (ctreatma) [inline internal/users/retrieve.go]: This should be wired up to the `--include` and `--exclude` CLI flags to match the behavior of the old code

File: internal/users/retrieve.go (hunk 1 above)
Comment (codinja1188) [reply internal/users/retrieve.go]: updated

File: internal/users/retrieve.go (hunk 2 above)
Comment (codinja1188) [reply internal/users/retrieve.go]: updated

Comment (ctreatma) [reply internal/users/retrieve.go]: This is still passing in `nil` for include and exclude; it should be making a call to the servicer in order to pull in the values for the corresponding CLI flags.

Comment (codinja1188) [reply internal/users/retrieve.go]: updated

File: internal/users/add.go [hunk 3]
```diff
@@ -46,20 +47,18 @@ func (c *Client) Add() *cobra.Command {
...
-				Invitee:     email,
-				Roles:       roles,
-				ProjectsIDs: projectsIDs,
-			}
-			invitation, _, err := c.InvitationService.Create(organizationID, createRequest, getOpts)
+			invitationInput := metal.NewInvitationInput(email)
+			invitationInput.SetRoles(roles)
+			invitationInput.SetProjectsIds(projectsIDs)
+
+			invitation, _, err := c.InvitationService.CreateOrganizationInvitation(context.Background(), organizationID).InvitationInput(*invitationInput).Execute()
```
Comment (ctreatma) [inline internal/users/add.go]: It looks like this API supports `include` parameters: https://github.com/equinix-labs/metal-go/blob/main/metal/v1/api_organizations.go#L219

To match what the old code was doing with `getOpts`, this needs to be wired up to the `--include` command line flags via the servicer.

**Review by ctreatma — APPROVED** [review]

**Pull request merged**